spec/intl.html

```diff
@@ -9,6 +9,9 @@ <h1>Amendments to the ECMAScript® 2021 Internationalization API Specification</
       This section lists amendments which must be made to <a href="https://tc39.es/ecma402/">ECMA-402, the ECMAScript® 2021 Internationalization API Specification</a>.
       Text to be added is marked <ins>like this</ins>, and text to be deleted is marked <del>like this</del>.
     </p>
+    <p>
+      This text is based on top of the ECMA-402 spec text from commit b5edf2cdcca8d69590796cd41abf8399d249b8a1.
```

Comment:
Maybe helpful to make the commit hash a link to github

Reply:
I think the hash text is enough for the case I included it `git diff <hash>` but why not?

Reply:
done